src/App.tsx

```diff
@@ -23,6 +24,11 @@ const App = () => {
             <Route path="/homepage" element={<Homepage />} />
           </Route>
         </Routes>
+        <Routes>
+          <Route element={<PrivateRoute />}>
```

Comment:
Is this normal to have a Route within a Route? I've never seen this pattern before